### PR TITLE
 Allow managed identity in COPY INTO

### PR DESCRIPTION
**Pull request by webrouse**
Jira: https://keboola.atlassian.net/browse/COM-731

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Writes data to Azure Synapse Database
   - `schema` - string (optional, default value `dbo`)
 - `tableId` - string (required)
 - `dbName` - string (required)
+- `absCredentialsType` - enum (optional): `sas` (default), `managed_identity`
```
Comment (webrouse) [inline README.md:20]: Vieme konfigurovat credentials typ, by default bez zmeny.

File: src/Adapter/AbsAdapter.php [hunk 2]
```diff
@@ -106,6 +109,26 @@ private function getEntries(): array
...
+    {
+        switch ($this->credentialsType) {
+            case self::CREDENTIALS_TYPE_SAS:
+                return sprintf(
+                    'IDENTITY=\'Shared Access Signature\', SECRET=\'?%s\'',
+                    $this->connectionAccessSignature
+                );
+
+            case self::CREDENTIALS_TYPE_MANAGED_IDENTITY:
+                return 'IDENTITY=\'Managed Identity\'';
```
Comment (webrouse) [inline src/Adapter/AbsAdapter.php:122]: Zmena v COPY INTO query.

File: tests/functional/error-managed-identity/expected-stderr [hunk 3]
```diff
@@ -0,0 +1 @@
+Query execution error: %AManaged Service Identity has not been enabled on this server. Please enable Managed Service Identity and try again.%A
```
Comment (webrouse) [inline tests/functional/error-managed-identity/expected-stderr:1]: Nemame to v nasej CI Synapse zapnute, ale mozeme aspon testovat, ze to neprejde.

File: src/Application.php [hunk 4]
```diff
@@ -76,7 +79,6 @@ private function processRunAction(array $tableConfig): array
                 $this->writeFullFromAdapter($tableConfig, $adapter);
             }
         } catch (UserException $e) {
-            $this['logger']->error($e->getMessage());
```
Comment (webrouse) [inline src/Application.php:79]: Chyby sa logovali duplicitne, tu aj v `run.php`.

**Review by ondrajodas — APPROVED** [review]
LGTM

**Pull request merged**